### DISCUSSION TITLE
App crashes whole OS

### DISCUSSION THREAD

**Discussion by Luciogi** [saber-notes/saber#360] [General]
### Steps to produce:

1.  Open Saber
2. Create page
3. Fill up some lines on page
4. Rotate device in landscape
5. Fully zoom in
6. Make Stroke about 10 points
7. Draw
8. It will freeze whole screen and crashes whole OS then system reboots or completely shutdown

### My observations:
 - It does not crashed in portrait mode
 
### Device Info:
- OS: Android 13 (crDroid)
- RAM 4 GB
- Device: Redmi 9T

### Tested on versions:
- 0.8.0 (Fdroid)
- 0.9.2 (Dirty, Foss_ALT)

Note: If anyone can produce this crash, i will convert this discussion to Issue(bug)

**Comment by adil192** [comment]
This is likely a bug with your ROM. Its maintainer may be able to help you

**Reply by Luciogi** [reply]
I will test with GSI image

Edit: GSI not working

**Reply by Luciogi** [reply]
Openned issue for rom https://github.com/crdroidandroid/issue_tracker/issues/134

**Comment by Luciogi** [comment]
@adil192 I guess it was flutter problem. Now I am using Saber 0.9.4, crash is not produce-able anymore
